### PR TITLE
Fixed Sailthru signal bug

### PR DESCRIPTION
**Pull request by clintonb**
ECOM-5955

### CODE REVIEW

**Comment by clintonb** [comment]
@edx/ecommerce please review

File: ecommerce/sailthru/signals.py [hunk 1]
```diff
@@ -127,10 +125,10 @@ def _build_course_url(course_id):
...
-    except BasketAttributeType.DoesNotExist:
-        attribute_type = BasketAttributeType.objects.create(name=SAILTHRU_CAMPAIGN)
-    return attribute_type
+def get_basket_attribute_type():
+    """ Returns `BasketAttributeType` for Sailthru campaign ID.
+
+    Returns:
+        BasketAttributeType
+    """
+    return BasketAttributeType.objects.get(name=SAILTHRU_CAMPAIGN)
```
Comment (rlucioni) [inline ecommerce/sailthru/signals.py:134]: Why is it that we can get rid of the exception handler? Do we always expect there to be a matching `BasketAttributeType`?

**Review by rlucioni — APPROVED** [review]
Thank you very much for addressing this.

File: ecommerce/sailthru/signals.py (hunk 1 above)
Comment (clintonb) [reply ecommerce/sailthru/signals.py:134]: The handler seems to have been added before we had a migration creating the `BasketAttributeType`.

Comment (rlucioni) [reply ecommerce/sailthru/signals.py:134]: Got it, thanks.

**Pull request merged**